tests/cover/test_reflection.py

```diff
@@ -225,6 +225,10 @@ def test_source_of_lambda_is_pretty():
         lambda x: True
     ) == 'lambda x: True'  # pragma: no cover
 
+def test_bracket_whitespace_is_striped():
+    assert get_pretty_function_description(
+        lambda x: ( x + 1 )
+    ) == 'lambda x: (x + 1)'  # pragma: no cover
```

Comment:
You probably meant to take this bit out when you added it to test_\lambda\_formatting.py